### PR TITLE
moved stuff around and added more text

### PR DESCRIPTION
**Pull request by AaronWatters**
I put things in my preferred order and added more explanation.  I also removed `commit --amend` since
you shouldn't need to do that unless you have done things in the wrong order.

### CODE REVIEW

**Comment by kostyat** [comment]
Hey Aaron, the changes look good, but why did you remove the whole part about `git pull`, `git remote add` etc.? That part was very useful for me because I'm not going to remember how to do that unless it's in a document.

File: devnotes.md [hunk 1]
```diff
@@ -49,13 +72,48 @@ For me, this was:
...
+`git remote add upstream https://github.com/simonsfoundation/inferelator_ng.git`
+
+## Merging changes from the remote into your fork (as needed)
+
+To merge changes from the main remote repository into your fork 
+[github suggests](https://help.github.com/articles/syncing-a-fork) the following
+sequence
+
+```
+# Fetch the upstream changes
```
Comment (AaronWatters) [inline devnotes.md:91]: I changed the procedure for pulling from the remote to match the work flow documented in the github documentation.

**Comment by AaronWatters** [comment]
See line note here:
https://github.com/simonsfoundation/inferelator_ng/pull/6/files#diff-616f127e2e07f9fe995adf5cc0afa82eR91
-- snd thanks because I actually omitted an important step which I just
fixed.
Let me know if you have other comments or suggestions.

  -- Aaron Watters

**Comment by kostyat** [comment]
Thank you, I think that's much better! Merging now.

**Pull request merged**